WordPress/Classes/Services/JetpackScanService.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+@objc class JetpackScanService: LocalCoreDataService {
+    private lazy var service: JetpackScanServiceRemote = {
+        return JetpackScanServiceRemote(wordPressComRestApi: defaultApi())
+    }()
+
+    @objc func getScanAvailable(for blog: Blog, success: @escaping(Bool) -> Void, failure: @escaping(Error) -> Void) {
+        guard let siteID = blog.dotComID?.intValue else {
+            return
+        }
+
+        service.getScanAvailableForSite(siteID, success: success, failure: failure)
+    }
+
+    private func defaultApi() -> WordPressComRestApi {
```

Comment:
We have this in so many places that I wonder if we should extend `WordPressComRestApi` and add that as a `static` func.

We don't need to change the other places that this is used but it would be good to have a centralized place.

Reply:
Done in: 7282ac7b7f40c1cfa6577dd48e92bd704016b6b5